packages/playwright-core/src/server/fetch.ts

```diff
@@ -517,7 +517,11 @@ function toHeadersArray(rawHeaders: string[]): types.HeadersArray {
 const redirectStatus = [301, 302, 303, 307, 308];
 
 function parseCookie(header: string): types.NetworkCookie | null {
-  const pairs = header.split(';').filter(s => s.trim().length > 0).map(p => p.split('=').map(s => s.trim()));
+  const pairs = header.split(';').filter(s => s.trim().length > 0).map(p => {
+    const value = p.slice(p.indexOf('=') + 1).trim();
+    const key = p.slice(0, p.indexOf('=')).trim();
```

Comment:
This will not work for the attributes that have only name (no value and no '='), e.g. `HttpOnly`. Please fix this and extend your test case to cover this.

Reply:
Thank you for the fast review! I will!

Do you have a tip for me where I can execute this test only? Running the full test suite locally causes a lot of test failures. Thank you :)

Reply:
Nevermind, this does the trick:

```
npm run ctest -- tests/global-fetch-cookie.spec.ts
```

Reply:
Would it be acceptable to export `parseCookie` and test it individually? It seems a bit difficult to test it well given how it is integrated

Reply:
We normally don't do unit tests. You can read parsed cookie via `request.storageState()`, similar to this test: https://github.com/microsoft/playwright/blob/6a7a2971f2e40442d720f42412d299c504fcbd71/tests/global-fetch-cookie.spec.ts#L175-L188

Reply:
Thank you, that was very helpful!